hathor/p2p/states/ready.py

```diff
@@ -180,3 +200,48 @@ def handle_pong(self, payload: str) -> None:
         self.ping_min_rtt = min(self.ping_min_rtt, self.ping_rtt)
         self.ping_start_time = None
         self.log.debug('rtt updated', rtt=self.ping_rtt, min_rtt=self.ping_min_rtt)
+
+    def send_get_best_blockchain(self, nBlocks: str = '10') -> None:
+        """ Send a GET-BEST-BLOCKCHAIN command, requesting a list of latest
+        N blocks from the best blockchain.
+        """
+        self.send_message(ProtocolMessages.GET_BEST_BLOCKCHAIN, nBlocks)
+
+    def handle_get_best_blockchain(self, payload: str) -> None:
+        """ Executed when a GET-BEST-BLOCKCHAIN command is received.
+        It just responds with a list N blocks from the best blockchain.
+        """
+        nBlocks = int(payload)
```

Comment:
Capture the exception and close the connection if payload is not an int, otherwise it will generate an unhandled exception.

Reply:
Thanks!